groups/forms.py

```diff
@@ -22,7 +22,7 @@ def build_helper(self):
         helper.layout = Layout(
             'body',
             FormActions(
-                Submit('comment-submit', 'Post comment'),
+                StrictButton('Post comment', type='submit'),
```

Comment:
Yes it should -- but that's one for another PR, I think, as there will surely be other places that need fixing too